QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeOneTote.java

```diff
@@ -0,0 +1,32 @@
+package org.team1540.quasarhelios;
+
+import ccre.channel.FloatInputPoll;
+import ccre.holders.TuningContext;
+import ccre.instinct.AutonomousModeOverException;
+
+public class AutonomousModeOneTote extends AutonomousModeBase {
+	protected FloatInputPoll toteDistance;
+	protected FloatInputPoll autoZoneDistance;
+	protected FloatInputPoll returnDistance;
+	
+	public AutonomousModeOneTote() {
+		super("One Tote");
+	}
+
+	@Override
+	protected void runAutonomous() throws InterruptedException,
+			AutonomousModeOverException {
+		drive(toteDistance.get());
+		QuasarHelios.autoLoader.set(true);
```

Comment:
While I know this isn't done, do we want to have a wait to make sure that it finishes loading?